portal/apps/org.wso2.carbon.iam.user-portal.app/src/main/pages/recovery/username.hbs

```diff
@@ -0,0 +1,82 @@
+                        <div class="data-container">
+                            <form class="form-horizontal" method="post">
+                                <h3>{{i18n "recover.username"}}</h3>
+                                {{#if isUsernameRecoveryDisabled}}
+                                        <div>
+                                            <h3>{{i18n "username.recovery"}}</h3>
+                                            <p>{{i18n "recovery.not.enabled"}}</p>
+                                            <div class="row">
+                                                <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12">
+                                                    <a href="{{@contextPath}}/" class="add-margin-top-2x">
+                                                        {{i18n "back.to.sign.in"}}</a>
+                                                </div>
+                                            </div>
+                                        </div>
+                                {{else}}
+                                <p>{{i18n "fill.below.to.recover"}}</p>
+                                <div class="form-group add-margin-top-5x">
+                                     {{#each usernameRecoveryClaims}}
+                                         {{log this}}
+                                            <div class="col-xs-12 col-sm-12 col-md-5 col-lg-5">
+                                                <label class="control-label"
+                                                       for={{claimLabel}}>{{displayName}}</label>
+                                                <input type={{#if dataType}}{{dataType}}{{else}}"text"{{/if}} name={{claimURI}} id={{claimLabel}}
+                                                                                                              class="form-control" placeholder={{displayName}}>
+                                            </div>
+                                     {{/each}}
+                                </div>
+                                <div class="form-group add-margin-top-5x">
+                                    <div class="col-xs-12 col-sm-12 col-md-5 col-lg-5">
+                                        <button type="submit" class="btn btn-default btn-primary add-margin-right-2x">{{i18n "recover"}}</button>
+                                        <a href="{{@contextPath}}/" class="hidden-xs hidden-sm">{{i18n "back.sign.in"}}</a>
+                                        <!--<a href="/login" class="visible-xs visible-sm add-margin-2x text-center">Back to sign in</a>-->
+                                    </div>
+                                </div>
+                                {{/if}}
+                            </form>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </div>
+
+{{/fillZone}}
```

Comment:
Add a new line.

Reply:
Fixed with https://github.com/wso2/product-is/pull/1371